README.md

```
@@ -1,61 +1,352 @@
-5. Add redirect URL's for `http://localhost:3000/oauth/callback` and `http://localhost:3000/oauth/guild`
-6. Run a local instance of Skyra, be sure to follow it's local development guidelines as well.
+[Official Site][official-site] · [Blog][blog] · [WolfStar Invite
+Link][invite-link] · [Support Server][discord-link] ·
+[Feedback][github-issues-link]
 
--   You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
+<!-- SHIELD GROUP -->
 
-7. Install dependencies with `yarn`.
-8. Run `yarn dev` to start the NextJS dev server
+[![][netlify-status-shield]][netlify-status-link]
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)